pkg/template/controller/metrics.go

```diff
@@ -0,0 +1,71 @@
+package controller
+
+import (
+	"time"
+
+	templateapi "github.com/openshift/origin/pkg/template/apis/template"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/apimachinery/pkg/labels"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+func newTemplateInstancesTotal() *prometheus.GaugeVec {
+	return prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "TemplateInstanceController_TemplateInstances_total",
+			Help: "Counts TemplateInstance objects by condition type and status",
+		},
+		[]string{"type", "status"},
+	)
+}
+
+func newTemplateInstancesWaiting() prometheus.Histogram {
+	return prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "TemplateInstanceController_TemplateInstances_active_waiting_time_seconds",
```

Comment:
Whether this remains a histogram or becomes a constant metric per the PR discussion thread most likely would *NOT* affect the name.

So still offering name suggestions prior to reaching consensus on that point seems OK :-)

I think `openshift_template_active_wait_time_seconds` would be good.